backends/gdx-backend-lwjgl/src/com/badlogic/gdx/backends/lwjgl/LwjglGraphics.java

```diff
@@ -79,7 +80,7 @@
 	}
 
 	LwjglGraphics (Canvas canvas, LwjglApplicationConfiguration config) {
-		this.config = config;
+		this(config);
```

Comment:
I think I originally changed something in the constructor, then got rid of the change. But I guess I left the `this` call in. It looks like it's functionally the same, though? Seems more directionally correct to re-use constructor code rather than duplicate it, right?